apps/emqx_durable_storage/src/emqx_ds.erl

```diff
@@ -154,6 +158,24 @@ open_db(DB, Opts = #{backend := Backend}) when Backend =:= builtin orelse Backen
     persistent_term:put(?persistent_term(DB), Module),
     ?module(DB):open_db(DB, Opts).
 
+-spec add_generation(db()) -> ok.
+add_generation(DB) ->
+    case persistent_term:get(?persistent_term(DB), undefined) of
+        undefined ->
+            ok;
```

Comment:
I think it's better to return an error or throw an exception if the database doesn't exist.